content/build/ci/github-actions/attestations.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Add SBOM and provenance attestations with GitHub Actions
+description: Add SBOM and provenance attestations to your images with GitHub Actions
+keywords: ci, github actions, gha, buildkit, buildx, attestations, sbom, provenance, slsa
+---
+
+SBOM and provenance [attestations](../../attestations/_index.md) add valuable
+metadata about the contents of your image, and how it was built. Adding these
+attestations to your images with GitHub Actions is as simple as adding two
+input fields: `provenance` and `sbom`.
```

Comment:
I suggest we also mention that you must be using v4+ of the build push action as I dont think this works in v3 and below.